### DISCUSSION TITLE
Quartz.net does not work when dockerized

### DISCUSSION THREAD

**Discussion by kibo-raghu** [quartznet/quartznet#918] [General]
I am using Quartz.net 3.0.7 in .net core 3.1 console app and hosting the app using IHostedService. I built the docker image and created containers but I don't see any quartz.net code working. Below is the docker file I used:

```
FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-buster-slim AS base
WORKDIR /app
EXPOSE 80

FROM mcr.microsoft.com/dotnet/core/sdk:3.1-buster AS build
WORKDIR /src
COPY IHostedSample.csproj ./
RUN dotnet restore "./IHostedSample.csproj"
COPY . .
WORKDIR "/src/."
RUN dotnet build "IHostedSample.csproj" -c Release -o /app/build

FROM build AS publish
RUN dotnet publish "IHostedSample.csproj" -c Release -o /app/publish

FROM base AS final
WORKDIR /app
COPY --from=publish /app/publish .


ENTRYPOINT ["dotnet", "IHostedSample.dll"]**
```

Will Quartz.net work when we dockerize the .net core app? If possible I need a working sample.

**Comment by lahma** [comment]
I don't see a reason why it wouldn't work. I think you should enable logging to console and see what kind of log messages you get.

**Reply by kibo-raghu** [reply]
I enabled logging before and after I call the scheduler classes and I dont see any errors as well. All I see is the custom console messages I used to track the code flow.. 

When deployed to kubernetes I see messages like "[40m[32minfo[39m[22m[49m: IHostedSample" the place I used Quartz.net classes in the logs.

**Reply by kibo-raghu** [reply]
Can you point me to a working sample if it worked?

**Reply by lahma** [reply]
I haven't done kubernetes but if you get the console app running on your machine via docker I would assume it would work the same way when deployed. You should enable logging for Quartz (maybe even DEBUG level), you get insights to what is happening and how Quartz is making decisions.

https://www.quartz-scheduler.net/documentation/quartz-3.x/tutorial/configuration-resource-usage-and-scheduler-factory.html#logging
